src/windows.vbs

```diff
@@ -32,7 +32,9 @@ sub createFile()
   objLink.WorkingDirectory = strCwd
   objLink.IconLocation = strIcon
   objLink.WindowStyle = strWindowMode
-  objLink.Hotkey = strHotkey
+  If Trim(strHotkey) <> "" Then
```

Comment:
We should add a comment here.

```suggestion
  ' We must skip setting the hotkey if the trimmed value is an empty string to avoid errors (#96).
  If Trim(strHotkey) <> "" Then
```